openspace.cfg

```diff
@@ -45,7 +45,8 @@ SGCTConfig = sgct.config.single{}
 
 -- Sets the scene that is to be loaded by OpenSpace. A scene file is a description
 -- of all entities that will be visible during an instance of OpenSpace
-Asset = "default"
+Asset = "asteroids"
```

Comment:
```suggestion
Asset = "default"
```

Reply:
Combining this step with next code change